localisation/00_wc_random_artifact_names.csv

```diff
@@ -37,55 +37,55 @@ artifact_jewelry_uncommon_prefix_martial_6;Raider's;;;;;;;;;;;;;x
 artifact_jewelry_uncommon_prefix_martial_7;Savage's;;;;;;;;;;;;;x
 artifact_jewelry_uncommon_prefix_martial_8;Squire's;;;;;;;;;;;;;x
 artifact_jewelry_uncommon_prefix_martial_9;Soldier's;;;;;;;;;;;;;x
-artifact_jewelry_uncommon_prefix_martial_10;Warrior's;;;;;;;;;;;;;x
+artifact_jewelry_uncommon_prefix_martial_10;Warrior's;;Kriegerisch;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
-artifact_jewelry_uncommon_prefix_stewardship_1;Mercantile;;;;;;;;;;;;;x
+artifact_jewelry_uncommon_prefix_stewardship_1;Mercantile;;Kaufm�nnisch;;;;;;;;;;;x
 artifact_jewelry_uncommon_prefix_stewardship_2;Caretaker's;;;;;;;;;;;;;x
 artifact_jewelry_uncommon_prefix_stewardship_3;Carpenter's;;;;;;;;;;;;;x
 artifact_jewelry_uncommon_prefix_stewardship_4;Craftsman's;;;;;;;;;;;;;x
-artifact_jewelry_uncommon_prefix_stewardship_5;Energizing;;;;;;;;;;;;;x
-artifact_jewelry_uncommon_prefix_stewardship_6;Expensive;;;;;;;;;;;;;x
+artifact_jewelry_uncommon_prefix_stewardship_5;Energising;;Anregend;;;;;;;;;;;x
```

Comment:
Pls, don't change English loc without notifying me, otherwise all your corrections may be simply deleted if I update this part, but don't make corrections to naming spreadsheet.